plutus-core/executables/src/PlutusCore/Executable/Parsers.hs

```diff
@@ -104,7 +104,7 @@ printmode = option auto
   <> showDefault
   <> help
     ("Print mode for textual output (ignored elsewhere): Classic -> plcPrettyClassicDef, "
-     <> "Debug -> plcPrettyClassicDebug, "
+     <> "Simple -> plcPrettyClassicSimple, "
```

Comment:
Why the renaming?

Reply:
The names changed in the prettyprinting code a while ago and `plcPrettyClassicDebug` no longer exists but the help message didn't get updated.